plugins/janus_videoroom.c

```diff
@@ -1916,6 +1948,7 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 		packet.timestamp = ntohl(packet.data->timestamp);
 		packet.seq_number = ntohs(packet.data->seq_number);
 		/* Go */
+		janus_mutex_lock(&participant->subscribers_mutex);
```

Comment:
While this check is probably needed, I'm afraid it might affect performance, considering how often it's called...